### ISSUE TITLE
Unable to download the WCAG Conformance Statement and the Section 508 VPAT for SharePoint Server 2016

### ISSUE THREAD

**Issue by EmilyYVChan** [MicrosoftDocs/OfficeDocs-SharePoint#982]
I tried to follow the instructions ( "click any of these links and search for SharePoint Server 2016" ) but couldn't find it. Please help. 

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2fa85ae0-dd91-b9b0-44a4-1e567b3a1a11
* Version Independent ID: 802553de-4d77-a634-1743-8d39e766ff30
* Content: [Accessibility guidelines in SharePoint Server 2016](https://docs.microsoft.com/en-gb/SharePoint/accessibility-guidelines)
* Content Source: [SharePoint/SharePointServer/accessibility-guidelines.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/SharePoint/SharePointServer/accessibility-guidelines.md)
* Product: **sharepoint-server-itpro**
* GitHub Login: @bentoncity
* Microsoft Alias: **stevhord**

**Comment by bentoncity** [comment]
Fixed links and republished.
